usaspending_api/awards/tests/unit/test_subawards.py

```diff
@@ -88,7 +96,7 @@ def test_specific_award(mock_matviews_qs):
         " chips cosby echo etsy forage future helvetica kale occupy salvia sartorial semiotics skateboard squid"
         " williamsburg yr. 8-bit banh beer before they sold out craft ethnic fingerstache fixie irony jean shorts"
         " life organic park photo booth retro salvia tattooed trade vhs williamsburg.",
-    "action_date": "2017-09-30",
+    "action_date": "2017-09-29",
```

Comment:
Curious why this date was changed

Reply:
so it doesn't tie with the other date. Might be a good idea to test sorting the the case of ties, but I'm lazy.